src/Microsoft.AspNet.JsonPatch/project.json

```diff
@@ -15,7 +15,10 @@
                 "System.Collections": "4.0.10-beta-*",
                 "System.Collections.Concurrent": "4.0.10-beta-*",
                 "System.Globalization": "4.0.10-beta-*",
-                "System.Runtime.Extensions": "4.0.10-beta-*"
+                "System.Runtime.Extensions": "4.0.10-beta-*",
+                "System.ComponentModel.TypeConverter": "4.0.0-beta-*",
+                "System.Reflection.Extensions": "4.0.0-beta-*",
+                "System.Text.Encoding.Extensions": "4.0.10-beta-*"
```

Comment:
That's what you're worried about? MVC doesn't even work right now :stuck_out_tongue_closed_eyes: